components/ui/header.tsx

```diff
@@ -40,9 +38,18 @@ export default function Header() {
                   About
                 </Link>
               </li>
+              <li>
+                <Link
+                  href="/signin"
+                  className="font-medium text-purple-600 hover:text-gray-200 px-4 py-3 flex items-center transition duration-150 ease-in-out"
+                >
+                  Signin
+                </Link>
+              </li>
             </ul>
           </nav>
-          {process.env.NODE_ENV !== 'production' && <AuthButton />} {/* AuthButton is hidden in production */} 
+          {process.env.NODE_ENV !== "production" && <AuthButton />}
+          {/* AuthButton is hidden in production */}
```

Comment:
We already have the auth buttons implemented. This change is not needed.

Reply:
No problem

Reply:
Please update Issues, so other contributors don't think It's still an issue @Ellahinator